lib/beaker-hostgenerator/data.rb

```diff
@@ -520,82 +400,6 @@ def osinfo
             'template' => 'cisco-exr-9k-x86_64',
           },
         },
-        'cumulus25-64' => {
-          :general => {
-            'platform'           => 'cumulus-2.5-x86_64',
-            'packaging_platform' => 'cumulus-2.2-amd64',
-          },
-          :vmpooler => {
-            'template' => 'cumulus-vx-25-x86_64',
-          },
-        },
-        'debian6-32' => {
-          :general => {
-            'platform' => 'debian-6-i386',
-          },
-          :vmpooler => {
-            'template' => 'debian-6-i386',
-          },
-        },
-        'debian6-64' => {
-          :general => {
-            'platform' => 'debian-6-amd64',
-          },
-          :vmpooler => {
-            'template' => 'debian-6-x86_64',
-          },
-        },
-        'debian7-32' => {
-          :general => {
-            'platform' => 'debian-7-i386',
-          },
-          :vmpooler => {
-            'template' => 'debian-7-i386',
-          },
-        },
-        'debian7-64' => {
-          :general => {
-            'platform' => 'debian-7-amd64',
-          },
-          :docker => {
-            'docker_image_commands' => [
-              'cp /bin/true /sbin/getty',
-              'apt-get update && apt-get install -y cron locales-all net-tools wget',
-            ],
-          },
-          :vagrant => {
-            'box' => 'debian/wheezy64',
-          },
-          :vmpooler => {
-            'template' => 'debian-7-x86_64',
-          },
-        },
-        'debian8-32' => {
-          :general => {
-            'platform' => 'debian-8-i386',
-          },
-          :vmpooler => {
-            'template' => 'debian-8-i386',
-          },
-        },
-        'debian8-64' => {
-          :general => {
-            'platform' => 'debian-8-amd64',
-          },
-          :docker => {
-            'docker_image_commands' => [
-              'cp /bin/true /sbin/agetty',
-              'rm -f /usr/sbin/policy-rc.d',
-              'apt-get update && apt-get install -y cron locales-all net-tools wget apt-transport-https',
-            ],
-          },
-          :vagrant => {
-            'box' => 'debian/jessie64',
-          },
-          :vmpooler => {
-            'template' => 'debian-8-x86_64',
-          },
-        },
         'debian9-32' => {
```

Comment:
I'm not sure if we still support debian 9? @pmcmaw @chelnak

Reply:
Yeah we no longer do, however some other teams may still, as for removal of OSs in the PR as a whole, I think it aligns well with what we currently support.

Reply:
I removed Debian 8 because it's gone from the Debian repos. I wasn't sure about Debian 9, so decided to be a bit more conservative. https://wiki.debian.org/LTS suggests the LTS ended 2022-06-30.

Reply:
We dropped agent support for Debian 9 in https://tickets.puppetlabs.com/browse/PA-4570. Feel free to remove it.